### ISSUE TITLE
More identicon flavors

### ISSUE THREAD

**Issue by Slesarew** [paritytech/parity-signer#877]
We need new visually distinctive identicons!

- [ ] metadata
- [ ] certificates (could be same as keys since they are indeed same)
- [ ] network specs

**Comment by Slesarew** [comment]
This could be moved to later milestone, but we need to decide: either we use current identicons and then replace them with other flavor, or use none until we have a replacement.

**Comment by goldsteinsveta** [comment]
Let's use the current ones. I can design _something_, but given this element is used in 99% of applications interacting w substrate-based chains, I think it should be thought through properly. I'd ask for an input from graphic designers too.

**Comment by Slesarew** [comment]
Query math geeks for algorithm too.

**Comment by Tbaut** [comment]
My 2 cents here, Identicons are only useful if they are a standard. If every app create their own, it's not helping anyone any more. I'd look at what Identicons are used in dapps, extensions etc, and go with this.

**Comment by goldsteinsveta** [comment]
Agree, parity-signer is not a target here. It's better to setup a separate repo

**Comment by Slesarew** [comment]
I want every identicon flavor in it's own repo! Compartmentalizing them is trivial and they should be re-used as much as possible.

Re-using something existing is a good approach indeed, but it's all about having pretty things, metadata check will be quite unique thing and if it does not look familiar - that's all better. Still, identicon is actually part of crypto flow, so re-implementing it just for fun is not exactly safe approach; we are discussing it only because it's such a soft human-involving matter.

**Comment by kirushik** [comment]
Well, at the same time some Signer-specific identities (like metadata/network specs) don't have any representation in any other interface so far, so we can go wild with designing (and specifying+modularizing!) new identicon systems for those usecases.

It might also turn out to be beneficial for UX/UI: having different _kinds_ of things having different, visually distinct identicon systems might help avoiding confusion.

**Comment by goldsteinsveta** [comment]
It is Signer-specific, but it does not appear only on Signer's GUI. Metadata/network specs are also relevant for any metadata update portal, that could be maintained by anyone in the ecosystem. I also hear requests for scripts to generate own metadata
